db/db.go

```diff
@@ -98,26 +98,41 @@ func GetEth1DepositsJoinEth2Deposits(query string, length, start uint64, orderBy
 	for _, column := range columns {
 		if orderBy == column {
 			hasColumn = true
+			break
 		}
 	}
 	if !hasColumn {
 		orderBy = "block_ts"
 	}
 
-	var err error
+	var searchLikeHash = regexp.MustCompile(`^0?x?[0-9a-fA-F]{2,96}`) // only search for pubkeys if string consists of 96 hex-chars
```

Comment:
Please compile regex outside of the  handler/function

Reply:
Did it in the last commit.